README.md

```diff
@@ -24,34 +24,48 @@ More detailed requirements are outlined in the [Installing the Islandora Essenti
 
+### Protecting the 'anonymous' username
+The `islandora_drupal_filter` passes the username of 'anonymous' through to Fedora for unauthenticated Drupal Users. A user with the name of 'anonymous' may have XACML policies applied to them that are meant to be applied to Drupal users that are not logged in or vice-versa. This is a potential security issue that can be plugged by creating a user named 'anonymous' and restricting access to the account. If this is done after installing Islandora, Drupal's cron can be run to remove expired authentication tokens.
+
+
 ## Configuration
 
-The `islandora_drupal_filter` passes the username of 'anonymous' through to Fedora for unauthenticated Drupal Users. A user with the name of 'anonymous' may have XACML policies applied to them that are meant to be applied to Drupal users that are not logged in or vice-versa. This is a potential security issue that can be plugged by creating a user named 'anonymous' and restricting access to the account.
+Configuration that applies to all solution packs, including the location of the Fedora Repository, the namespaces accessible by this instance of Islandora, and whether to generate derivatives on ingest, are available at  `admin/islandora/configure`.
+
+![Configuration](https://user-images.githubusercontent.com/1943338/40320855-724afcba-5d03-11e8-9109-0b8413349839.png)
 
-Drupal's cron can be run to remove expired authentication tokens.
+### Breadcrumb Generation
 
-**Breadcrumb Generation** on the configuration page, allows you to choose the default breadcrumb generation
- or a custom method (if implemented).
+Whether Drupal breadcrumbs (showing an object's parent hierarchy) should be displayed, and how they are generated, can be set on the configuration page. Other modules (such as [Islandora Solr](https://github.com/Islandora/islandora_solr_search)) may provide  alternatives that perform better at large scales than the built-in Resource Index query.
+
+### Inactive and Deleted Objects
+
+By default, objects with the [Fedora state](https://wiki.duraspace.org/display/FEDORA38/Fedora+Digital+Object+Model) of "Inactive" or "Deleted" are accessible to all users with the Drupal permission "View repository objects". It is possible to use a separate permission to control access to these non-"Active" objects, but this permission must first be enabled at `admin/islandora/configure`, then the permssion can be granted to desired roles at `admin/people/permissions`.
 
 ### Customization
 
-[Customize ingest forms](https://github.com/Islandora/islandora/wiki/Multi-paged-Ingest-Forms)
+Hooks provided by Islandora are documented in `islandora.api.php`. A [detailed tutorial](https://github.com/Islandora/islandora/wiki/Multi-paged-Ingest-Forms) on extending the multi-page ingest forms is available on the Github (developers') Wiki.
```

Comment:
Could we make this a two liner? It can lead people to believe that hooks and the tutorial are dependants? And seems to be both are 2 points worth mentioning in a larger list of possible the customization?